source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```
@@ -0,0 +1,2875 @@
+    font-family: UnRaid;
+  }
+  
+  .dockerSearch {
+    display: inline-block;
+    float: right;
+  }
+  
+  .donate {
+    color: var(--support-popup-text) !important;
+    background: var(--support-popup-background);
+    background: -webkit-linear-gradient(top, transparent 0%, rgba(0,0,0,0.4) 100%),-webkit-linear-gradient(left, lighten(var(--donate-background), 15%) 0%, var(--donate-background) 50%, lighten(var(--donate-background), 15%) 100%);
```

Biome (1.9.4)

[error] Unexpected unknown function: lighten

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)